desc/basis.py

```diff
@@ -2092,3 +2092,180 @@ def body(alpha, out):
     # loop which will execute necessary n values.
     out = fori_loop(0, (M_max + 1).astype(int), body, (out))
     return out
+
+
+def get_basis_poincare(X_lmn_3D, basis_3D, zeta=0):
```

Comment:
are these new functions needed?

Reply:
Yes, I use get_basis_poincare for setting lambda at the cross-section. Before xsection changes, I was also using FourierZernike_to_FourierZernike_no_N_modes, but I might delete it since I don't think I am using it anymore.

Reply:
ok id definitely vote for removing things that arent strictly necessary to reduce the amount of code we have to maintain